drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -279,7 +279,7 @@ function DragNDropTemplates(url_name) {
                     problemHeader,
                     h('p', {innerHTML: ctx.problem_html}),
                 ]),
-                h('section.drag-container', { attributes: { role: 'application' } }, [
+                h('section.drag-container', { attributes: { } }, [
```

Comment:
@pomegranited Any specific reason to keep the `attributes: {}` portion of the second argument to `h`?